src/utils/string/StringUtils.js

```diff
@@ -399,6 +399,11 @@ const onOpenLink = (url) => {
   });
 };
 
+const isValidUrl = (url) => {
+  const urlPattern = /^(https?:\/\/)?([\da-zA-Z.-]+)\.([a-zA-Z]{2,6})([/\w.-]*)*\/?$/;
+  return urlPattern.test(url);
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `isValidUrl` function uses a regular expression to validate URLs. However, this regular expression is quite simple and may not cover all valid URL cases. For example, it doesn't support internationalized domain names (IDNs) or URLs with query parameters. Consider using a more comprehensive regular expression or a library like `validator.js` which has a robust URL validation method.
```diff
-  const urlPattern = /^(https?:\/\/)?([\da-zA-Z.-]+)\.([a-zA-Z]{2,6})([/\w.-]*)*\/?$/;
+  const urlPattern = /^(https?:\/\/)?((([a-z\d]([a-z\d-]*[a-z\d])*)\.)+[a-z]{2,}|((\d{1,3}\.){3}\d{1,3}))(\:\d+)?(\/[-a-z\d%_.~+]*)*(\?[;&a-z\d%_.~+=-]*)?(\#[-a-z\d_]*)?$/i;
   return urlPattern.test(url);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->